rust/cymbal/src/frames/records.rs

```diff
@@ -109,10 +104,11 @@ impl ErrorTrackingStackFrame {
         // and so when we load a frame record we need to patch back up the context onto the frame,
         // since we dropped it when we serialised the frame during saving.
         let mut frame: Frame = serde_json::from_value(found.contents)?;
-        let context = if let Some(context) = found.context.as_ref() {
+
+        let context = if let Some(context) = found.context {
             // We serialise the frame context as a json string, but it's a structure we have to manually
             // deserialise back into the frame.
-            Some(serde_json::from_str(context)?)
+            serde_json::from_value(context)?
```

Comment:
As above, this will effectively parse the `json::null` values as well as `context IS NULL` rows, and I think we actually want to specifically only handle either cases where the contents of the `context` column are an actual json structure or a postgres NULL (the `Some()` wrapping previously here ensures that)